src/extension/wps/wps-core/src/main/java/org/geoserver/wps/ppio/WFSPPIO.java

```diff
@@ -129,14 +132,16 @@ public Object decode(Object input) throws Exception {
      */
     private SimpleFeatureCollection eliminateFeatureBounds(SimpleFeatureCollection fc) {
         final SimpleFeatureType original = fc.getSchema();
+        
         List<String> names = new ArrayList<String>();
         boolean alternateGeometry = false;
         for(AttributeDescriptor ad : original.getAttributeDescriptors()) {
             final String name = ad.getLocalName();
-            if(!"boundedBy".equals(name) && !"metadataProperty".equals(name)) {
+
+            if(!boundedBy.equals(name) && !metaData.equals(name)) {
```

Comment:
It was just a typo, but I thought moving to use constants would reduce the "magic"